priv/templates/nimble_template/test/otp_app_web/views/api/error_view_test.exs

```diff
@@ -0,0 +1,96 @@
+defmodule <%= web_module %>.Api.ErrorViewTest do
+  use <%= web_module %>.ConnCase, async: true
+
+  # Bring render/3 and render_to_string/3 for testing custom views
+  import Phoenix.View
+
+  alias <%= web_module %>.Api.ErrorView
+
+  defmodule Device do
+    use Ecto.Schema
+
+    import Ecto.Changeset
+
+    schema "devices" do
+      field :device_id, :string
+      field :operating_system, :string
+      field :device_name, :string
+
+      timestamps()
+    end
+
+    def changeset(device \\ %__MODULE__{}, attrs) do
+      device
+      |> cast(attrs, [
+        :device_id,
+        :operating_system,
+        :device_name
+      ])
+      |> validate_required([
+        :device_id,
+        :operating_system,
+        :device_name
+      ])
+    end
+  end
+
+  test "renders 404.json" do
+    assert render(ErrorView, "404.json", []) == %{
+             errors: [%{code: :not_found, detail: %{}, message: "Not Found"}]
+           }
+  end
+
+  test "renders 500.json" do
+    assert render(ErrorView, "500.json", []) == %{
+             errors: [
+               %{code: :internal_server_error, detail: %{}, message: "Internal Server Error"}
+             ]
+           }
+  end
+
+  test "renders custom error message" do
+    assert render(ErrorView, "500.json", status: 500, message: "Something went wrong") ==
+             %{
+               errors: [
+                 %{code: :internal_server_error, detail: %{}, message: "Something went wrong"}
+               ]
+             }
+  end
+
+  test "given error code and an invalid changeset has multiple errors fields, renders error.json" do
```

Comment:
Maybe 🤔 

```suggestion
  test "given error code and an invalid changeset with multiple errors fields, renders error.json" do
```

Reply:
yes, updated in https://github.com/nimblehq/elixir-templates/pull/140/commits/e355bb4b1264d6ff57a02f3ebf736394b52781d3